langwatch/src/components/messages/MessagesTable.tsx

```diff
@@ -1098,7 +1098,9 @@ export function MessagesTable() {
                       ? "calc(100vw - 580px)"
                       : showFilters
                       ? "calc(100vw - 450px)"
-                      : "50%"
```

Comment:
nitpic: could this be extracted to a function to make it clearer? Nested ternary operators, I find at least, are so hard to read and always end up causing bugs.